server/src/main/java/io/spine/server/procman/ProcessManager.java

```diff
@@ -121,43 +122,47 @@ protected B getBuilder() {
     /**
      * Transforms the passed list of event messages into the list of events.
      *
-     * @param eventMessages event messages for which generate events
-     * @param envelope the context of the command which generated the event messages
+     * @param  eventMessages event messages for which generate events
+     * @param  origin        the envelope with the origin of events
      * @return list of events
      */
-    private List<Event> toEvents(List<? extends Message> eventMessages,
-                                 final CommandEnvelope envelope) {
-        return CommandHandlerMethod.toEvents(getProducerId(),
-                                             getVersion(),
-                                             eventMessages,
-                                             envelope);
+    private List<Event> toEvents(List<? extends Message> eventMessages, MessageEnvelope origin) {
+        return HandlerMethod.toEvents(getProducerId(), getVersion(), eventMessages, origin);
     }
 
     /**
-     * Dispatches an event to the event subscriber method of the process manager.
+     * Dispatches an event to the event reactor method of the process manager.
      *
-     * @param event the envelope with the event
+     * @param  event the envelope with the event
+     * @return a list of produced events or an empty list if the process manager does not
+     *         produce new events because of the passed event
      */
-    void dispatchEvent(EventEnvelope event)  {
+    List<Event> dispatchEvent(EventEnvelope event)  {
```

Comment:
Being package-private is quite safe for now, but I agree: let's to return to this matter in a while.